Source/Core/DolphinQt/Config/Mapping/WiimoteEmuGeneral.cpp

```diff
@@ -92,14 +94,17 @@ void WiimoteEmuGeneral::OnAttachmentChanged(int extension)
   SaveSettings();
 }
 
-void WiimoteEmuGeneral::LoadSettings()
+void WiimoteEmuGeneral::Update()
 {
   auto* ce_extension = static_cast<ControllerEmu::Extension*>(
       Wiimote::GetWiimoteGroup(GetPort(), WiimoteEmu::WiimoteGroup::Extension));
 
   m_extension_combo->setCurrentIndex(ce_extension->switch_extension);
-  OnAttachmentChanged(ce_extension->switch_extension);
```

Comment:
This explicit call to OnAttachmentChanged here wasn't necessary as calling m_extension_combo->setCurrentIndex just above it ends up calling OnAttachmentChanged.